Mendelugram/Mendelugram/ViewController/Detail/PhotoDetailViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  PhotoDetailViewController.swift
+//  Mendelugram
+//
+//  Created by Pacek on 21/04/2019.
+//
+
+import UIKit
+
+class PhotoDetailViewController: UIViewController {
+
+    var photo: Photo!
+
+    @IBOutlet weak var tableView: UITableView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        title = "\(photo.author.username)'s photo"
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.register(UINib(nibName: PhotoTableViewCell.reuseIdentifier, bundle: nil), forCellReuseIdentifier: PhotoTableViewCell.reuseIdentifier)
+        tableView.register(UINib(nibName: CommentTableViewCell.reuseIdentifier, bundle: nil), forCellReuseIdentifier: CommentTableViewCell.reuseIdentifier)
+        tableView.register(UINib(nibName: PhotoDescriptionTableViewCell.reuseIdentifier, bundle: nil), forCellReuseIdentifier: PhotoDescriptionTableViewCell.reuseIdentifier)
+        tableView.delegate = self
+        tableView.dataSource = self
+    }
+
+}
+
+extension PhotoDetailViewController: UITableViewDataSource {
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 3
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return section <= 1 ? 1 : photo.comments.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        switch indexPath.section {
+        case 0:
```

Comment:
How about having an enum with Int raw values where you would have sections nicely named?